src/app/data-model-provider/Provider.h

```diff
@@ -79,6 +79,14 @@ class Provider : public ProviderMetadataTree
     ///    - validation of ACL/timed interaction flags/writability, if those checks are desired.
     virtual ActionReturnStatus WriteAttribute(const WriteAttributeRequest & request, AttributeValueDecoder & decoder) = 0;
 
+    ///   Indicates the start/end of a series of list operations. This function will be called either before the first
+    ///   Write operation or after the last one of a series of consequence attribute data of the same attribute.
+    ///
+    ///   1) This function will be called if the client tries to set a nullable list attribute to null.
+    ///   2) This function will only be called once for a series of consequent attribute data (regardless the kind of list
```

Comment:
Again, "consequent" does not seem like the right word, but I can't tell what this is actually trying to say.

Reply:
Changed the description, from what I understand those are "consecutive" writes.